README.md

```diff
@@ -31,6 +31,7 @@ They implement the following public methods:
         * `messageTypeField` - which field in the message describes the type of a message. This field needs to be defined as `z.literal` in the schema and is used for resolving the correct schema for validation
         * `locatorConfig` - configuration for resolving existing queue and/or topic. Should not be specified together with the `creationConfig`.
         * `creationConfig` - configuration for queue and/or topic to create, if one does not exist. Should not be specified together with the `locatorConfig`.
+        * `messageTimestampField` - which field in the message contain the message creation date. This field needs to be ISO-8601 date string and by default it is `timestamp`, if your messages doesn't contain it the library will add one automatically to avoid infinite loops on consumer;
```

Comment:
can we support Date too? nothing prevents zod schema from transforming timestamps within the schema

Reply:
Already done 🙇